packages/ui/lit-grid/src/playwright/dx-grid.spec.ts

```diff
@@ -0,0 +1,217 @@
+//
+// Copyright 2021 DXOS.org
+//
+
+import { expect, test, type Browser, type Page, type Locator } from '@playwright/test';
+
+import { setupPage, storybookUrl } from '@dxos/test-utils/playwright';
+
+import { type DxGridCellsSelect, type DxGridPlanePosition } from '../types';
+import { toPlaneCellIndex } from '../util';
+
+const gridPlaneCellSize = 31;
+const gap = 1;
+const nCols = 9;
+const nRows = 7;
+
+class GridManager {
+  constructor(browser: Browser) {
```

Comment:
I'd recommend just passing in a locator to the manager class and doing things like `setupPage` outside of the class because it makes it more composable. This will work fine for this suite likely but couldn't be easily reused higher up the stack.